src/utils/staking.js

```diff
@@ -421,6 +425,9 @@ export class Staking {
         } else {
             lockupId = getLockupAccountId(accountId)
         }
+        if (this.wallet.accountExists(lockupId)) {
```

Comment:
@mattlockyer this looks like never tested. Missing `await`. Also missing `!`. Basically seems like it would always throw as `accountExists` would always return a not null promise.